tests/routes/desktop_browser/test_push_triggers.py

```
@@ -0,0 +1,608 @@
+            # At this point, the critical path (chat broadcast) would have already
+            # returned. The push may or may not have started yet depending on
+            # scheduling, but the task is queued.
+            assert not send_finished.is_set(), (
+                "push.send should not have completed yet — it takes 0.5s"
+            )
+            # Clean up by cancelling
+            task.cancel()
+            try:
+                await task
+            except (asyncio.CancelledError, Exception):
+                pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)